### PR TITLE
[WEB-4929] Chrome Extension is not pulling all of the information on Zillow and rent comp map crashing

### PR DESCRIPTION
**Pull request by VeronikaVasilchikova**
https://bowery.atlassian.net/browse/WEB-4929

### CODE REVIEW

**Review by igor-kazakevich — APPROVED** [review]

File: src/parsers/ZillowParser.js [hunk 1]
```diff
@@ -27,14 +27,27 @@ export default class ZillowParser {
   }
 
   get rent() {
-    const [rent] = $('.ds-home-details-chip').text().match(priceRegExp()) || [null]
-    return extractNumber(rent)
+    const container = this.document.querySelector('.data-column-container .summary-container')
+    const containerSpanNodes = container.querySelectorAll('span')
+    if (containerSpanNodes.length) {
+      let rentIndex
```
Comment (mrromro) [inline src/parsers/ZillowParser.js:33]: Could we use `Array.prototype.findIndex()` instead?

File: src/parsers/ZillowParser.js [hunk 2]
```diff
@@ -27,14 +27,27 @@ export default class ZillowParser {
   }
 
   get rent() {
-    const [rent] = $('.ds-home-details-chip').text().match(priceRegExp()) || [null]
-    return extractNumber(rent)
+    const container = this.document.querySelector('.data-column-container .summary-container')
```
Comment (mrromro) [inline src/parsers/ZillowParser.js:30]: I do not like jQuery's `$` but I think it would be better not to mix `document.query...` and `$`

**Review by mrromro — APPROVED** [review]
It looks good.

File: src/parsers/ZillowParser.js (hunk 1 above)
Comment (VeronikaVasilchikova) [reply src/parsers/ZillowParser.js]: I improve this part of code with latest commit. Please, take a look

File: src/parsers/ZillowParser.js (hunk 2 above)
Comment (VeronikaVasilchikova) [reply src/parsers/ZillowParser.js]: I improved this part of code with latest commit. Please, take a look

**Review by mrromro — APPROVED** [review]

**Pull request merged**